src/canister/individual_user_template/src/api/cdao/mod.rs

```diff
@@ -104,13 +102,16 @@ async fn deploy_cdao_sns(
         let registered = matches!(cdata.session_type, Some(SessionType::RegisteredSession));
         (registered, cdata.cdao_canisters.len() == CDAO_TOKEN_LIMIT)
     });
-    /*if !registered {
-        return Err(CdaoDeployError::Unregistered);
-    }*/
+
     if limit_hit {
         return Err(CdaoDeployError::TokenLimit(CDAO_TOKEN_LIMIT));
     }
 
+    // Alloting 0.5T more to the user canister to be on safer side while deploying canisters
+    request_cycles_from_subnet_orchestrator(6 * USER_SNS_CANISTER_INITIAL_CYCLES)
```

Comment:
Why are we multiplying by 6 here ?

Reply:
There are 5 canisters so 0.5T each and allocating 0.5T more as a buffer for performing operations